content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,34 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.16 [2020-08-06]
+
+### Breaking
```

Comment:
We also need to add that the `influx repl` command was removed. I'm adding the following warning to the `influx repl` doc:

```
{{% warn %}}
#### Removed in InfluxDB 2.0 beta-16
The `influx repl` command was removed in **InfluxDB 2.0 beta-16**.
To use the Flux REPL, build the REPL from source.
For more information, see the [Flux GitHub repository](https://github.com/influxdata/flux/#readme).
{{% /warn %}}
```